### ISSUE TITLE
Print Button not working 

### ISSUE THREAD

**Issue by subhashprajapat** [stephanrauh/ngx-extended-pdf-viewer#1303]
we are clicking on print button so i am getting error.

```
**Unhandled Promise rejection: Cannot set properties of null (setting 'textContent') ; Zone: <root> ; Task: Promise.then ; Value: TypeError: Cannot set properties of null (setting 'textContent')
    at PDFPrintService.destroy (viewer-2.14.442.min.js:22:301197)**
```
    
using code 
   
```html
<ngx-extended-pdf-viewer  [src]="'https://vadimdez.github.io/ng2-pdf-viewer/assets/pdf-test.pdf'" [useBrowserLocale]="true" [showPrintButton]="true">
</ngx-extended-pdf-viewer>
```

**Comment by stephanrauh** [comment]
Sounds like you're using the latest alpha version. You're right, I removed the print area accidentally.

**Comment by stephanrauh** [comment]
Your bugfix has landed with version 13.0.0-alpha.8.

Enjoy!
Stephan

**Comment by subhashprajapat** [comment]
i am using version 13.0.0-alpha.8. now working fine 👍 
Thanks 

Subhash
